src/index.scss

```diff
@@ -92,29 +92,18 @@ body {
       height: 100%;
       mix-blend-mode: multiply;
     }
-
-    .drawing-cancelation-button {
-      pointer-events: auto;
-      position: sticky;
-      bottom: 1rem;
-      left: 1rem;
-      font-size: 1rem;
-      max-width: 10rem;
-      text-align: left;
-      outline: 0;
-      padding: 0.5rem;
-      margin: 0;
-      border: 1px solid black;
-      border-radius: 0.25rem;
-      background: #fff;
-
-      &:hover {
-        background: #eee;
-      }
-    }
```

Comment:
some of the css here is because the existing popup is a button so it needed overrides for the default browser attributes (like `outline: 0` or `text-align: left`).

now that it's a div we don't need some of them but I'd still prefer to have the `max-width` and the `border` and `border-radius`, so that it's more noticeable on the page